movie-app/src/components/movie-item/MovieItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import MovieItemWrapper from "../../components/movie-item/MovieItemWrapper";
+import posterLogo from "../../assets/images/netflix-streaming-vs-traditional-cable.jpg";
+import { IMovie } from "../../interfaces/Movie";
+
+const MovieItem = (props: IMovie) => {
+  return (
+    <MovieItemWrapper
+      onClick={() =>
```

Comment:
In React we usually try to avoid to create a listener function on each render, because it makes impossible to make child components `pure`. Consider using `useCallback` instead

Reply:
Fixed